viper/modules/pe.py

```diff
@@ -254,7 +258,8 @@ def get_compiletime(pe):
         if not self.__check_session():
             return
 
-        compile_time = get_compiletime(self.pe)
+        self.result_compile_time = get_compiletime(self.pe)
+        compile_time = self.result_compile_time
```

Comment:
The reason why I introduced "self.result_compile_time" is that I wanted to access the "raw" result of calling pe.PE().run() (with argmument "compiletime"). Parsing the output from stdout is possible but not really nice. I also stumbled over the fact that the compile_time is printed in **bold**. 

In my opinion it would be useful to have a general concept of how the results of the modules can and should be accessed. I'm thinking along the lines of adding a `self.result = None` (or ` self.result = {}`) to the Module class. Then when an instance runs any code it should store the result in that variable/dict. This would be useful for testing and also for the API/Web interface.

But this is not really related/tied to the Python3 effort.. so should I open this as a Issue/PR on the Viper master?